src/model.rs

```diff
@@ -463,6 +466,37 @@ impl Collections {
     }
 }
 
+impl AddPrefix for Collections {
```

Comment:
I wonder if this is the good place

Reply:
Would you do it in `src/add_prefix.rs` as I did for `Collection` and `CollectionWithId`?

Reply:
I moved the implementation of `AddPrefix` for `Collections` into `add_prefix.rs` in fb0858d854a6075ad80dfc16aa95fdd9f76e0943.